WalletWasabi.Backend/wwwroot/about.html

```diff
@@ -86,11 +86,13 @@
                     <div class="col-md-8">
                         <h1 class="title-lined left">Motivation</h1>
                         <p>
-                            We believe that privacy is the biggest problem in Bitcoin today. Wasabi Wallet was created to help Bitcoin maintain it's cypherpunk roots in it's path to global reserve currency and to fix some of the inherent privacy issues of an open and public ledger. After years of doing privacy research and development, we've concluded that having good tools is not enough - They need to be easy to use or they will only be utilized by minority, making users stand out in the crowd. This is why Wasabi is designed to be user friendly and to provide privacy automatically by default. User only needs to worry about sending and receiving.
+                            We believe, that the most urgent problem for mankind is fixing the money, that Bitcoin is our best option for doing it and that privacy is its biggest problem.
                         </p>
-                        <p>We live in a world where your information is being used to typecast, manipulate and surveil you. This is why Wasabi Wallet is an open-source, non-custodial, privacy-focused Bitcoin wallet for desktop that implements trustless coinjoin magic. The development of Wasabi Wallet is sponsored by zkSNACKs who hires the people and determines the vision for the future.</p>
                         <p>
-                            Being put in a position to trust someone is always uncomfortable, especially regarding personal information or your money. For this reason Wasabi Wallet is trustless by design. We don’t, we haven’t and we never will collect any information from you. What you do with your Bitcoin is your business so use Wasabi Wallet to retain your individuality, regain your identity and ultimately, reclaim your privacy. We let you control what you reveal about your Bitcoin life.
+                            Wasabi Wallet was created to help Bitcoin maintain the cypherpunk roots, in its path to a global reserve currency and to mitigate the inherent privacy issues of its public ledger. This is why Wasabi Wallet is an open-source, non-custodial, privacy-focused Bitcoin wallet for desktop that implements trustless coinjoin magic. 
+                        </p>
+                        <p>
+                            After years of doing privacy research and development, we've concluded that having good tools is not enough - They need to be easy to use or they will only be utilized by minority, making users stand out in the crowd. To increase the use of these tools, Wasabi is designed to be user friendly and to provide privacy automatically by default.
```

Comment:
```suggestion
                            After years of doing privacy research and development, we've concluded that having good tools is not enough - They need to be easy to use otherwise they will only be utilized by minority, making users stand out in the crowd. To increase the use of these tools, Wasabi is designed to be user friendly and to provide privacy automatically by default.
```